### PR TITLE
Fix: Updated nsqlookupd protocol client

### PR DESCRIPTION
**Pull request by fclairamb**
Tiny change to make it compatible with the [spec](http://nsq.io/clients/building_client_libraries.html#discovery) and the actual nsqlookupd server.

I guess the "address" field must come from a previous version and "somebody" must still use it.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>ly.bit</groupId>
     <artifactId>nsq</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>0.0.1-fclairamb</version>
```
Comment (mreiferson) [inline pom.xml:5]: can we drop this change

File: src/main/java/ly/bit/nsq/NSQReader.java [hunk 2]
```diff
@@ -142,7 +142,7 @@ public void addLookupd(AbstractLookupd lookupd) {
 		if (stored != null){
 			return;
 		}
-        lookupdScheduler.scheduleAtFixedRate(new BasicLookupdJob(addr, this), 30, 30, SECONDS);
+        lookupdScheduler.scheduleAtFixedRate(new BasicLookupdJob(addr, this), 5, 30, SECONDS);
```
Comment (mreiferson) [inline src/main/java/ly/bit/nsq/NSQReader.java:145]: and this?

**Comment by fclairamb** [comment]
Hum, I feel stupid. 

**Short version**: Only the first commit is right.

**Explanation**:
I made the "nsq-java-fork" repo to have a version _only for me_ (the 0.0.1-fclairamb) because I thought you might take some time to accept my changes. But I was wrong (you are fast) and I managed to add my "private" commits by mistake.

File: src/main/java/ly/bit/nsq/NSQReader.java (hunk 2 above)
Comment (fclairamb) [inline src/main/java/ly/bit/nsq/NSQReader.java:145]: yes

File: pom.xml (hunk 1 above)
Comment (fclairamb) [inline pom.xml:5]: yes

**Comment by fclairamb** [comment]
Replaced it by #12.

**Pull request closed without merging**